lib/buttons-and-links/KExternalLink.vue

```diff
@@ -72,19 +72,29 @@
         let styles = {};
         if (this.openInNewTab) {
           if (this.isRtl) {
-            // If RTL-language, but English link, displays correct margins
             styles['marginRight'] = '8px';
             if (this.text !== this.href) {
               styles['marginRight'] = '0px';
               styles['marginLeft'] = '8px';
             }
           } else {
-            styles['marginLeft'] = '8px';
+            styles['marginRight'] = '8px';
           }
         }
         return { ...styles };
       },
+      /**
+       * Changes icon direction according to language of text of URL
+       */
+      iconStyle() {
+        let styles = { top: '4px' };
+        if (this.text === this.href) {
```

Comment:
I think that we can remove this `iconStyle` computed property although I totally see why you added it.

I switched my Firefox to Arabic to confirm this, but people who are viewing RTL content would probably just expect the icon's arrow always to point left because that's where their new tab would open (note where the + is along the tabs at the top):

![image](https://user-images.githubusercontent.com/6356129/103708054-a21c0000-4f64-11eb-9e79-c8faf9993448.png)

So even if the link's text is English or another LTR language, if the page is displayed in RTL (aka `this.isRtl === true`) then the icon should always be flipped as if the text associated is RTL even if it looks kinda funny next to some LTR text.

---

As for the margins, that looks good to me!

Reply:
Following up on the above - some of this also kind of confirms my thoughts about the icon I think: https://material.io/design/usability/bidirectionality.html#mirroring-layout (at least the bit about leftward arrows meaning forward motion to RTL readers)

Reply:
That makes sense! Fixed.